### ISSUE TITLE
Left and right hand tracking at the same time

### ISSUE THREAD

**Issue by atx-barnes** [Reality-Dev/BodyTracking#4]
Would this be possible to track both hands at the same time instead of just one? If so do you have a demo that shows this functionality? I've been able to test the example project out and it's amazing, great work.

Thanks!

**Comment by Reality-Dev** [comment]
Hey there!
After reading this I worked on it. I just pushed a new update to the package to enable tracking more than one hand at a time. There is an example inside of `ARSUIViewHandTracking2D` that uses this new functionality. If you select "Show Hand Tracking 2D" in the example app, it will now track two hands. To add more hand trackers, all you must do is initialize a new tracker like this: `HandTracker2D(arView: self)` (and I recommend maintaining a strong reference to it somewhere and calling `destroy()` whenever you are done with it). For it to be visible on screen, you must attach views to its joints with `handTracker.attach(thisView:, toThisJoint:)`.
I'm glad you like the example project! It's there to help you and the rest of the community. I am working on a complete Body Tracking with ARKit & RealityKit course that I hope will help you even more!
